index.js

```diff
@@ -40,3 +40,8 @@ module.exports.butWhatActuallyIsCinnamonToastCrunchThough =
       "just might be a cereal involving cinnamon, toast, and something " +
       "crunchy!"
   }
+
+module.exports.butWhyIsCinnamonToastCrunchGood = 
+  function nobodyEvahExplainedItToMeToo () {
```

Comment:
Since you mentioned tweaking the function names, I like the export as-is, but how about this for the function's own name?

```suggestion
  function nobodyEvahExplainedThisToMeEither () {
```